dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -584,7 +588,9 @@ boolean isUnusedClassSymbolReference(ClassSymbolReference reference) {
           constantPoolIndexForClass(sourceJavaClass, targetClassName);
       Verify.verify(
           !targetConstantPoolIndices.isEmpty(),
-          "The target class symbol reference is not found in source class");
+          "The reference to the target class %s is not found in the source class %s",
```

Comment:
Is this accurate? The source class is missing a reference? I'd expect something like

Source class A references target class B, but B could not be found. 

This is more like 

Target class B could be found but source class A does not have a reference to target class B.

Reply:
Clarified the message. This `verify` is asserting "This should not happen".